src/CONST.js

```diff
@@ -399,7 +399,20 @@ const EXPENSIFY_EMAILS = [
     CONST.EMAIL.BILLS, CONST.EMAIL.STUDENT_AMBASSADOR, CONST.EMAIL.QA_TRAVIS,
 ];
 
+const EXCLUDED_GROUP_EMAILS = [
```

Comment:
Can you please put one entry per line in each of these for readability's sake? You'd be my hero if you wanted to do the same to the `EXPENSIFY_EMAILS` array too.

Reply:
I'll do that :) Have you verified if the emails in the list are correctly mapped?

Reply:
I have and they all seem good. Can you add one more address to all three lists? `svfg@expensify.com`

Reply:
Noted. Will do.

Reply:
Done